local/kind/setup.sh

```diff
@@ -69,6 +69,13 @@ for cluster in "${CLUSTERS[@]}"; do
         KIND_EXPERIMENTAL_PROVIDER=${KIND_EXPERIMENTAL_PROVIDER:-} ${KIND_CMD} create cluster \
             --config "${TMP_DIR}/${cluster}.config" \
             --kubeconfig "${TMP_DIR}/${cluster}.kubeconfig"
+
+	if [[  ${KIND_CMD} == "sudo kind" ]]; then
+		sudo chmod +r "${TMP_DIR}/${cluster}.kubeconfig"
+	fi
+
+	printf "Provisioning ingress router in ${cluster}\n"
```

Comment:
Why use of `printf` over `echo` for simple strings?

Reply:
[more standardized/consistent](. https://unix.stackexchange.com/questions/65803/why-is-printf-better-than-echo). I try to favor printf over echo.